cli/src/highlight.rs

```diff
@@ -252,27 +266,68 @@ fn style_to_css(style: ansi_term::Style) -> String {
         write!(&mut result, "font-style: italic;").unwrap();
     }
     if let Some(color) = style.foreground {
-        write!(&mut result, "color: {};", color_to_css(color)).unwrap();
+        write_color(&mut result, color);
     }
     result.push('\'');
     result
 }
 
-fn color_to_css(color: Color) -> &'static str {
-    match color {
-        Color::Black => "black",
-        Color::Blue => "blue",
-        Color::Red => "red",
-        Color::Green => "green",
-        Color::Yellow => "yellow",
-        Color::Cyan => "cyan",
-        Color::Purple => "purple",
-        Color::White => "white",
-        Color::Fixed(n) => CSS_STYLES_BY_COLOR_ID[n as usize].as_str(),
-        _ => panic!("Unsupported color type"),
+fn write_color(buffer: &mut String, color: Color) {
```

Comment:
folded `color_to_css` into this since we can't return the computed RGB string as `&'static str`